### DISCUSSION TITLE
Zu viel Theorie bisher?

### DISCUSSION THREAD

**Discussion by wulfland** [wulfland/BootCampBasta23#3] [Polls]
❓

**Comment by mark-arn** [comment]
✌️

**Comment by Gorphi** [comment]
# Theorie  

Ich finde es passend uns ein wenig mit Theorie zu fluten, da man sich im normalen Arbeitsalltag eh nicht damit beschäftigt 💯 🥼 

> Cool finde ich auch dass man sofort Vorschläge bekommt, wenn man das # symbol schreibt.
#4
